compiler/fir/checkers/checkers-component-generator/src/org/jetbrains/kotlin/fir/checkers/generator/diagnostics/FirDiagnosticsList.kt

```diff
@@ -377,6 +377,7 @@ object DIAGNOSTICS_LIST : DiagnosticList() {
         val RETURN_TYPE_MISMATCH by error<KtExpression>(PositioningStrategy.WHOLE_ELEMENT) {
             parameter<ConeKotlinType>("expected")
             parameter<ConeKotlinType>("actual")
+            parameter<FirDeclaration>("targetDeclaration")
```

Comment:
Can this be more specific, e.g., `FirSimpleFunction`?

Reply:
Sure sounds good!